src/dev/impl/DevToys/ViewModels/Tools/Text/XmlValidator/XmlValidatorToolProvider.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Composition;
+using Windows.UI.Xaml.Controls;
+using DevToys.Api.Tools;
+using DevToys.Core.Threading;
+using DevToys.Shared.Api.Core;
+
+namespace DevToys.ViewModels.Tools.XmlValidator
+{
+    [Export(typeof(IToolProvider))]
+    [Name("Xml Validator")]
+    [Parent(TextGroupToolProvider.InternalName)]
+    [ProtocolName("xmlvalidator")]
+    [Order(1)]
+    [NotScrollable]
+    internal sealed class XmlValidatorToolProvider : ToolProviderBase, IToolProvider
+    {
+        private readonly IMefProvider _mefProvider;
+
+        public string MenuDisplayName => LanguageManager.Instance.XmlValidator.MenuDisplayName;
+
+        public string? SearchDisplayName => LanguageManager.Instance.XmlValidator.SearchDisplayName;
+
+        public string? Description => LanguageManager.Instance.XmlValidator.Description;
+
+        public string AccessibleName => LanguageManager.Instance.XmlValidator.AccessibleName;
+
+        public string? SearchKeywords => LanguageManager.Instance.XmlValidator.SearchKeywords;
+
+        public string? XmlData => LanguageManager.Instance.XmlValidator.XmlData;
```

Comment:
This property doesn't seem to be used. Should it be removed?

Reply:
Removed.